packages/stems/src/components/TokenValueInputV2/TokenValueInputV2.tsx

```diff
@@ -0,0 +1,110 @@
+import { useCallback, useMemo, useRef } from 'react'
+
+import BN from 'bn.js'
+import cn from 'classnames'
+import { uniqueId } from 'lodash'
+
+import { useControlled } from 'hooks/useControlled'
+
+import styles from './TokenValueInputV2.module.css'
+import { TokenValueInputV2Props } from './types'
+
+/**
+ * Format a number to have commas
+ */
+const formatNumberCommas = (num: number | string) => {
```

Comment:
do we want this in a common string util somewhere?

Reply:
It's in common already! Should I pull it in?

Reply:
YUH

Reply:
Possible side effect - I'm changing it so that if there's a `.` then it includes a `.` - even if there's no decimal digits

Reply:
I think that's probably ok! assuming this new behavior came with some design?

Reply:
Comes from not being able to type in a `.` without it. I think effectively this is the same?

Reply:
👍

Reply:
Eh, adding @audius/common to stems was a PITA. keeping this as-is for now until client folks can figure out the best way to integrate these.